server/openapi_server/controllers/physical_address_controller.py

```diff
@@ -16,6 +22,56 @@ def physical_addresses_read_all(note=None):  # noqa: E501
 
     :rtype: List[PhysicalAddressAnnotation]
     """
+    failure = {
+        "detail": "Unknown",
+        "status": 400,
+        "title": "Bad Request",
+        "type": "about:blank"
```

Comment:
This is interesting. Did you grab this example from somewhere?

Reply:
No Modeled it off what the spec said we should be returning.